### DISCUSSION TITLE
Elasticsearch error: Bad Gateway

### DISCUSSION THREAD

**Discussion by curtishall** [bsmithio/OPNsense-Dashboard#8] [Q&A]
I followed the instructions, but when adding Elasticsearch to Grafana I get this error:

Elasticsearch error: Bad Gateway

Any help would be appreciated.

**Comment by tonygauderman** [comment]
Is the elasticsearch container running, is the URL for the data source "http://elasticsearch:9200", and is the version set to 7.10+ ?
